orbit/pkg/update/update.go

```diff
@@ -106,19 +105,25 @@ func NewUpdater(opt Options) (*Updater, error) {
 	}
 
 	tufClient := client.NewClient(opt.LocalStore, remoteStore)
-	var rootKeys []*data.PublicKey
-	if err := json.Unmarshal([]byte(opt.RootKeys), &rootKeys); err != nil {
-		return nil, fmt.Errorf("unmarshal root keys: %w", err)
-	}
 
+	// First try initializing via stored local metadata. Older Orbit packages were distributed with
```

Comment:
Am a bit confused, older as in current stable version? or older than that?

AFAICS our generated packages (generated with `main` as of today) do contain the full `root.json`?